INSTALL.md

```diff
@@ -805,6 +805,13 @@ Build (and install) the FIPS provider
 Don't perform FIPS module run-time checks related to enforcement of security
 parameters such as minimum security strength of keys.
 
+### no-fips-post
+
+Dont perform FIPS module Power On Self Tests.
+
+This option should only be used for debugging only. It is useful when trying to
```

Comment:
I would use stronger words: `This option must be used for debugging only as it makes the fips provider non-compliant. It is useful when trying to add breakpoints when testing FIPS algorithms.`